lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -186,14 +186,30 @@ mapping:
           nodes that will run Galaxy jobs, unless using Pulsar.
 
       tool_config_file:
-        type: str
-        default: config/tool_conf.xml,config/shed_tool_conf.xml
+        type: list or str
```

Comment:
Just think you can makeup syntax as you go huh 😆?  ``KeyError: 'list or str'`` - this needs to be ``any`` I think. Config validation doesn't work at all on 19.09 though so I guess I can see why and it doesn't seem to have tests.

Reply:
I make it up, you implement it. It's a great relationship.